package/scripts/auth.py

```diff
@@ -13,7 +13,6 @@
 # the License.
 #
 
-import sys
 import ambari_helpers as helpers
 from resource_management import *
```

Comment:
From (http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html#importing):

To adapt a well-known exchange:

(Exterior Dagobah, jungle, swamp, and mist.)

LUKE: Is from ``module import *`` better than explicit imports?

YODA: No, not better. Quicker, easier, more seductive.

LUKE: But how will I know why explicit imports are better than the wild-card form?

YODA: Know you will when your code you try to read six months from now.

Wild-card imports are from the dark side of Python.